test/coverlet.collector.tests/CoverletSettingsParserTests.cs

```diff
@@ -42,17 +42,24 @@ public void ParseShouldSelectFirstTestModuleFromTestModulesList()
             Assert.Equal("module1.dll", coverletSettings.TestModule);
         }
 
-        [Fact]
-        public void ParseShouldCorrectlyParseConfigurationElement()
+        [Theory]
+        [InlineData("[*]*,[coverlet]*", "[coverlet.*.tests?]*,[coverlet.*.tests.*]*", @"E:\temp,/var/tmp", "module1.cs,module2.cs", "Obsolete,GeneratedCodeAttribute,CompilerGeneratedAttribute")]
+        [InlineData("[*]*, [coverlet]*", "[coverlet.*.tests?]*, [coverlet.*.tests.*]*", @"E:\temp, /var/tmp", "module1.cs, module2.cs", "Obsolete, GeneratedCodeAttribute, CompilerGeneratedAttribute")]
+        [InlineData("[*]*,\r\n[coverlet]*", "[coverlet.*.tests?]*,\r\n[coverlet.*.tests.*]*", "E:\\temp,\r\n/var/tmp", "module1.cs,\r\nmodule2.cs", "Obsolete,\r\nGeneratedCodeAttribute,\r\nCompilerGeneratedAttribute")]
+        public void ParseShouldCorrectlyParseConfigurationElement(string includeFilters,
```

Comment:
Can you add some test with empty space and null?

Reply:
Added a few cases for missing elements, null inner texts and whitespace

Reply:
thank's!